files/build_templates/qos_config.j2

```diff
@@ -214,6 +232,48 @@
         }
     },
 {% endif %}
+{% if 'resource_type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['resource_type'] in apollo_resource_types %}
+    "QUEUE": {
+{% for port in PORT_ACTIVE %}
+        "{{ port }}|3": {
+            "scheduler"   : "scheduler.1",
+            "wred_profile": "AZURE_LOSSLESS"
+        },
+{% endfor %}
+{% for port in PORT_ACTIVE %}
```

Comment:
since the only change in this table is for queue 4, add the if-else construct only for queue 4 so as to avoid repetition

Reply:
Thanks for suggestion. Fixed it.